packages/contracts/scripts/smt-checker/README.md

```diff
@@ -20,6 +20,7 @@ Once installed, you can verify that Z3 is correctly installed by checking the ve
 
 ## Step 3: Use Forge to Test Contracts Using SMT
 
+Check that you do not have a profile ready at foundry.toml you might skip this step, else run
```

Comment:
If I run update.ts with the default SMT profile it should work and update it like adding the contracts. Do you confirm it is the case ?

Reply:
You run update.ts, to add this profile to foundry.toml, but you asked to also added instead, so it is there, at https://github.com/ubiquity/ubiquity-dollar/pull/702/commits/69cedc429f34aee26a41171b41518e620b3b4267

update.ts create the profile, if it's not existent

You run smt.ts  (now scripted https://github.com/ubiquity/ubiquity-dollar/pull/702/commits/ab1a2b5b762b0520292b4d5d1d845c6dcf7fd590) to add contracts to the line "contracts" in this specific profile

Reply:
@zgorizzo69 the video is clear https://www.dropbox.com/s/wkgqjir8msrwxa1/full_smt.mp4?dl=0